sdk/storage/AzureStorageBlob/Source/Options/ClientMethodOptions.swift

```diff
@@ -432,13 +607,103 @@ public struct UploadBlobOptions: RequestOptions, Codable, Equatable {
         self.customerProvidedEncryptionKey = customerProvidedEncryptionKey
         self.customerProvidedEncryptionScope = customerProvidedEncryptionScope
         self.encoding = encoding
-        self.timeoutInSeconds = timeoutInSeconds
+        self.timeout = timeout
     }
 
     // TODO: Evalute whether serializing/deserializing dispatchQueue is necessary
     enum CodingKeys: CodingKey {
         case clientRequestId, cancellationToken, leaseAccessConditions, modifiedAccessConditions,
             encryptionOptions, customerProvidedEncryptionKey, customerProvidedEncryptionScope,
-            encoding, timeoutInSeconds
+            encoding, timeout
+    }
+}
+
+/// User-configurable options for the `BlobsOperations.getMetadata` operation.
+public struct GetBlobMetadataOptions: RequestOptions {
+    /// A client-generated, opaque value with 1KB character limit that is recorded in analytics logs.
+    public let clientRequestId: String?
+
+    /// A token used to make a best-effort attempt at canceling a request.
+    public let cancellationToken: CancellationToken?
+
+    /// A dispatch queue on which to call the completion handler. Defaults to `DispatchQueue.main`.
+    public var dispatchQueue: DispatchQueue?
+
+    /// A `PipelineContext` object to associate with the request.
+    public var context: PipelineContext?
+
+    /// An `Rfc1123Date` specifying the snapshot you wish to query.
+    public let snapshot: Rfc1123Date?
+
+    /// A n`Rfc1123Date` specifying the version of the blob to query.
+    public let versionId: Rfc1123Date?
+
+    /// The timeout parameter is expressed in seconds. This method may make
```

Comment:
Is this true? I would think GetMetadata would be a single operation.

Reply:
Removed. This was copy-paste from Upload/Download operations where this would be the case.